### PR TITLE
Expand section 2

### PR DESCRIPTION
**Pull request by navjotk**

### CODE REVIEW

**Comment by mlange05** [comment]
I remain highly suspicious that the first two items can actually be done in 10min without talking over people's head, but I guess we'll see once we have the code and text for this section.

Good to go from me.

**Comment by FabioLuporini** [comment]
Merged.

File: index.ipynb [hunk 1]
```diff
@@ -55,6 +55,17 @@
     "  * Exercise 2: Making a wave! **[5min]**\n",
     "\n",
     "* Session 2: [Seismic Imaging](seismic_imaging.ipynb)\n",
+    "    * Add receivers to record observations and build a complete forward operator **[5min]**\n",
+    "    * Exercise 3: Gradient operator **[5min]**\n",
+    "    * Intro to optimisation **[5min]**\n",
+    "        * Compare with machine learning, weather forecasting etc. minimise difference between observed and simulated.\n",
+    "        * Gradient descent. In a PDE constrained optimisation problem, the gradient is obtained by the adjoint-state method.\n",
```
Comment (mloubout) [inline index.ipynb:62]: This is not correct, adjoint-state is not a PDE constrained optimization problem.

Comment (navjotk) [reply index.ipynb:62]: "We will solve a PDE-constrained optimisation problem with a gradient-descent method, obtaining the gradient using the adjoint-state method."
Does the above rephrasing make you happier @mloubout ?

Comment (mloubout) [reply index.ipynb:62]: Well no, that's the thing, in time-domain, we only solve the  "reduced problem" which is an unconstrained least-square problem (or other depending on the way you measure the misfit). The PDE constraint is eliminated due to computationally unsolvable to have the reduced unconstrained adjoint-state problem.

Comment (mloubout) [reply index.ipynb:62]: So i would simply say
" We will solve an adjoint-state optimization problem with gradient descent for a simple seismic inversion problem"

**Pull request merged**